specification/automation/resource-manager/Microsoft.Automation/stable/2018-01-15/dscNodeConfiguration.json

```diff
@@ -443,7 +443,7 @@
           "$ref": "#/definitions/DscConfigurationAssociationProperty",
           "description": "Gets or sets the configuration of the node."
         },
-        "incrementNodeConfigurationBuild": {
+        "IncrementNodeConfigurationBuild": {
```

Comment:
Please update the corresponding fields in this example too: https://github.com/vrdmr/azure-rest-api-specs/blob/0489a8a2c0bf758e9593a7d21ff507e0899ff698/specification/automation/resource-manager/Microsoft.Automation/stable/2018-01-15/examples/createOrUpdateDscNodeConfiguration.json

Reply:
Fixed.